Source/JavaScriptCore/API/tests/testapi.mm

```diff
@@ -649,9 +649,8 @@ static void testObjectiveCAPIMain()
         checkResult(@"Should be a created from Obj-C", symbol.isSymbol);
     }
 
-// Older platforms ifdef the type of some selectors so these tests don't work.
-// FIXME: Remove this when we stop building for macOS 10.14/iOS 12.
-#if (defined(__MAC_OS_X_VERSION_MIN_REQUIRED) && __MAC_OS_X_VERSION_MIN_REQUIRED >= 101500) || (defined(__IPHONE_OS_VERSION_MIN_REQUIRED) && __IPHONE_OS_VERSION_MIN_REQUIRED >= 130000)
+// FIXME: These tests fail on tvOS and watchOS
```

Comment:
Is this something that you tested and confirmed? If not, it's better to not state this as a fact, but to say something like "Does this actually fail on watchOS and tvOS?" or "Need to confirm whether this fails on watchOS and tvOS."

I would expect that this condition was just wrongly written from the start, and then it became obvious after some mass replacement. Didn't check git history though.

Reply:
I tested and confirmed this.

Reply:
As in, I removed this check and ran all the tests and empirically determined this.

Reply:
Could you please tell us more? Did you run in simulator or on the device, and what was the failure?

Reply:
The failure was that the tests did not even build.

Reply:
error: cannot initialize a parameter of type 'JSValueProperty' (aka 'NSString *') with an lvalue of type 'JSValue *__strong'